src/rdatomic.h

```diff
@@ -25,7 +25,8 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-#pragma once
+#ifndef _RD_ATOMIC_H_
```

Comment:
to be consistent remove second underscore: `_RDATOMIC_H_`
This goes for the ones below as well.